.gitignore

```diff
@@ -3,4 +3,5 @@
 build
 dist
 upseto.egg-info
-.coverage
+.coverage*
```

Comment:
This existed before my time shlomp had coverage but some how i got new files that showed in git.